lib/__tests__/has-property.ts

```diff
@@ -8,45 +8,45 @@ describe('assert.dom(...).hasProperty()', () => {
   beforeEach(() => {
     assert = new TestAssertions();
 
-    document.body.innerHTML = '<input type="password">';
+    document.body.innerHTML = '<input type="checkbox">';
```

Comment:
any reason for replacing the existing test instead of adding a new one?

Reply:
I thought this way it would be more universal. But I can move it into a separate test if that’s preferred.

Reply:
> But I can move it into a separate test if that’s preferred.

yes, I think that would be better to ensure that the existing behavior doesn't break :)

Reply:
ok, updated.